docker-compose.yml

```diff
@@ -5,16 +5,25 @@ services:
     build:
```

Comment:
Not a diff here but introduced in #57, version on L1 isn't necessary and throws a warning when we run, we should remove. This was deprecated by docker since version 25.05: https://github.com/compose-spec/compose-spec/blob/main/spec.md#version-and-name-top-level-elements